### DISCUSSION TITLE
请教关于备份大文件夹的问题：我想知道使用WebDav或者对象存储进行备份时，是直接将文件夹进行打包上传吗？是否会打一个非常大的包？

### DISCUSSION THREAD

**Discussion by glxaa** [1Panel-dev/1Panel#4475] [Q&A]
目前我备份的文件只有25G，我看是直接打成了一个包进行上传，如果该文件夹具有200个G，那么是否会打成一个200G的包还是拆分为几个小包进行上传备份呢？据所知，很多网盘只支持不超过100G或者200G文件的上传，那么1panel的处理逻辑是什么？对于超大文件是拆分后分批次上传吗？

**Comment by wanghe-fit2cloud** [comment]
目前版本还没有拆分到操作。
